inc/nav-moh.php

```diff
@@ -1,39 +1,62 @@
 <?php
 /**
- * This partial contains site navigation for the music oral history resource.
+ * The template for displaying the child navigation bar.
  *
- * @link https://github.com/MITLibraries/music-oral-history
- *
- * @package WordPress
- * @subpackage music-oral-history theme
- * @since 1.1.11
+ * @package MIT_Libraries_Child
+ * @since 2.0.0
```

Comment:
This isn't the Child theme anymore so this should get updated.

Reply:
ok